app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ApplicationsController < ApplicationController
+  def new
+  end
+
+  def show
+    if params[:search].present?
+      @searched_pets = Pet.search(params[:search])
+    else
+      @searched_pets = []
+    end
+    @applicant = Applicant.retrieve_applicant(params[:id])
+    @pets = list_pets(@applicant)
+  end
+
+  def create
+    applicant = Applicant.new({
+      name: params[:name],
+      street_address: params[:street_address],
+      city: params[:city],
+      state: params[:state],
+      zip_code: params[:zip_code],
+      description: params[:description],
+    })
+
+    if applicant.save
+      application = PetsApplication.create!(applicant: applicant)
```

Comment:
I'm loving this transition from applicant to Application and look forward to seeing this work!